libs/user_extensions/src/TTAlpsHistogramFiller.cpp

```diff
@@ -1315,8 +1436,10 @@ void TTAlpsHistogramFiller::FillSingleMuonABCDHistograms(const shared_ptr<Event>
           {"phi", muon->Get("phi")},
  
           {"dxyPVTraj", muon->Get("dxyPVTraj")},
-          {"logDxyPVTraj", TMath::Log10(fabs(muon->Get("dxyPVTraj")))},
+          {"absDxyPVTraj", fabs(float(muon->Get("dxyPVTraj")))},
+          {"logDxyPVTraj", TMath::Log10(fabs(float(muon->Get("dxyPVTraj"))))},
```

Comment:
Here same as above. Maybe it's just easier if you use `GetAs<float>(...)` everywhere?